Anature/src/application/enums/Species.java

```diff
@@ -2,5 +2,6 @@
 
 public enum Species
 {
-	NotSet, Null
+	NotSet, Null, Sardino, Midaqua, Smoldren, Scorta, Scaldris, Sapron, Sodren, Arganian, Transgendeero, 
+	Transgendeera, Clouder, Cloutrain, Cloudoom, Belvidia
```

Comment:
You're missing a couple of Anatures:
Modenine, Routweiler, Fiberwire, Poire, Aquatecta